api/src/pcapi/connectors/beneficiaries/ubble.py

```diff
@@ -50,6 +51,14 @@ def _get_data_attribute(response: ubble_fraud_models.UbbleIdentificationResponse
     return response["data"]["attributes"].get(name)  # type: ignore [index]
 
 
+def _parse_ubble_gender(ubble_gender: str | None) -> users_models.GenderEnum | None:
+    if ubble_gender == "M":
+        return users_models.GenderEnum.M
+    if ubble_gender == "F":
+        return users_models.GenderEnum.F
+    return None
```

Comment:
Ou peut-être quelque chose plus en mode _"ask forgiveness not permission"_ ?

```python
try:
    users_models.GenderEnum[ubble_gender]
except KeyError:
    return None
```

ou encore

```python
# from contextlib import suppress

with suppress(KeyError):
    return users_models.GenderEnum[ubble_gender]
```

J'aurais bien suggéré `users_models.GenderEnum.get(ubble_gender)` mais la méthode n'existe pas pour les `Enum` :'(

Reply:
Ooh intéressant
J'avais pas fais attention que les srings correspondent aus clefs de l'enum

Je vais faire ça :+1: 
Je préfère le try:except explicite je pense.